pilot/pkg/proxy/envoy/v2/debug.go

```diff
@@ -48,8 +49,6 @@ func (s *DiscoveryServer) InitDebug(mux *http.ServeMux, sctl *aggregate.Controll
 	})
 
 	mux.HandleFunc("/debug/edsz", edsz)
-
-	mux.HandleFunc("/debug/ldsz", ldsz)
```

Comment:
Why remove ?

Reply:
Listeners are outputted as part of the adsz, this is inline with @rshriram's initial PR.